supernode/config/constants.go

```diff
@@ -75,3 +79,15 @@ const (
 	// SubsystemDfget represents metrics from dfget
 	SubsystemDfget = "dfget"
 )
+
+const (
+	// DefaultInitialDelay is the delay time from the start to the first GC execution.
+	DefaultInitialDelay = 6 * time.Second
+
+	// DefaultFixedDelay is the delay time to execute the next GC after the previous one is finished.
+	DefaultFixedDelay = 2 * time.Minute
+
+	// DefaultTaskExpireTime when a task is not accessed within the taskExpireTime,
+	// and it will be treated to be expired.
+	DefaultTaskExpireTime = 3 * time.Minute
```

Comment:
`DefaultFixedDelay` and `DefaultTaskExpireTime` should have flags. And `DefaultInitialDelay` can be a const value. WDYT?

Reply:
I think it is enough for them to be configured in the configuration file, and no need to have corresponding flags. WDYT?